src/dto/shared.dto.ts

```diff
@@ -32,6 +32,12 @@ export interface AddressBalanceFilters extends Pagination {
    * List of addresses to check.
    */
   addresses: string[]
+
+  /**
+   * Optional filter for token types. If not specified, all token types are returned.
+   * Allowed values are `native`, `fungible`, `nft` and `multitoken`.
+   */
+  tokenTypes?: string[]
```

Comment:
Would it be possible to instead of string to have a enum? Because you are stating that it will be only these values 
Allowed values are `native`, `fungible`, `nft` and `multitoken`.. 
and we are in typescript not in js :-D